lms/djangoapps/mobile_api/utils.py

```diff
@@ -1,31 +1,92 @@
 """
-Tests for video outline API
+Common utility methods and decorators for Mobile APIs.
 """
 
 
+import functools
+from django.http import Http404
+
+from opaque_keys.edx.keys import CourseKey
 from courseware import access
-from student.roles import CourseBetaTesterRole
-from student import auth
+from courseware.courses import get_course_with_access
+from rest_framework import permissions
+from rest_framework.authentication import OAuth2Authentication, SessionAuthentication
 
 
-def mobile_available_when_enrolled(course, user):
+def mobile_course_access(depth=0, verify_enrolled=True):
```

Comment:
It is being used by `VideoSummaryList` here: https://github.com/edx/edx-platform/pull/6398/files#diff-4bef877c8b70416409d4e1c5021cf45cR82